man/journal-remote.conf.xml

```diff
@@ -94,6 +120,7 @@
         <citerefentry><refentrytitle>systemd-journal-remote.service</refentrytitle><manvolnum>8</manvolnum></citerefentry>,
         <citerefentry><refentrytitle>systemd</refentrytitle><manvolnum>1</manvolnum></citerefentry>,
         <citerefentry><refentrytitle>systemd-journald.service</refentrytitle><manvolnum>8</manvolnum></citerefentry>
+        <citerefentry><refentrytitle>journald.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry>
```

Comment:
Please sort, and separate entries with `,`.

Reply:
Is there a preference for where - goes alphabetically?

Does journal-remote come before or after journald?  ASCII rules?